packages/fuels/tests/contracts/foo_caller_contract/src/main.sw

```diff
@@ -18,4 +19,8 @@ impl FooCaller for Contract {
 
         !response
     }
+
+    fn return_bool(value: bool) -> bool {
```

Comment:
Nit: use a different contract entirely to "add call that does not need ContractId". 
`foo_caller_contract` is very specific in what it does, having this method here seems out of place.